package.json

```diff
@@ -93,7 +96,26 @@
         "xml.fileAssociations": {
           "type": "array",
           "default": [],
+          "items": {
+            "type": "object",
+            "properties": {
+              "systemId": {
+                "type": "string",
+                "description": "The path or URL to the XML schema (XSD or DTD)"
+              },
+              "pattern": {
+                "type": "string",
+                "description": "File glob pattern. Example: **/*.Format.ps1xml",
+                "markdownDescription": "File glob pattern. Example: `**/*.Format.ps1xml`"
+              }
+            },
+            "required": [
+              "systemId",
+              "pattern"
+            ]
+          },
           "description": "Allows XML schemas to be associated to file name patterns.\n\nExample:\n[{\n  \"systemId\": \"path/to/file.xsd\",\n  \"pattern\": \"file1.xml\"\n},\n{\n  \"systemId\": \"path/to/file2.xsd\",\n  \"pattern\": \"file2.xml\"\n}]",
+          "markdownDescription": "Allows XML schemas to be associated to file name patterns.\n\nExample:\n```json\n[{\n  \"systemId\": \"path/to/file.xsd\",\n  \"pattern\": \"file1.xml\"\n},\n{\n  \"systemId\": \"path/to/file2.xsd\",\n  \"pattern\": \"file2.xml\"\n}]\n```",
```

Comment:
maybe use a glob pattern in file2 example

Reply:
Perhaps you could add a link to https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob

Reply:
agreed, that's a better link

Reply:
Updated.

Reply:
or perhaps this link https://github.com/angelozerr/lsp4xml/wiki/Configuration#file-associations

Reply:
I think it is good now